test/models/data_lake_store/test_data_lake_store_account.rb

```diff
@@ -0,0 +1,57 @@
+require File.expand_path '../../test_helper', __dir__
+
+# Test class for Data Lake Store Account Model
+class TestDataLakeStoreAccount < Minitest::Test
+  def setup
+    @service = Fog::DataLakeStore::AzureRM.new(credentials)
+    @account = data_lake_store_account(@service)
+    @account_client1 = @service.instance_variable_get(:@account_client)
+    @response = ApiStub::Models::DataLakeStore::DataLakeStoreAccount.create_data_lake_store_account_obj(@account_client1)
+  end
+
+  def test_model_methods
+    methods = [
+        :save,
+        :update,
+        :destroy
+    ]
+    methods.each do |method|
+      assert_respond_to @account, method
+    end
+  end
+
+  def test_model_attributes
+    attributes = [
+        :name,
+        :id,
+        :resource_group
+    ]
```

Comment:
Add all the other attributes that are present in the `DataLakeStoreAccount` model